lib/jsonapi/include_directive.rb

```diff
@@ -12,6 +12,8 @@ module JSONAPI
   # @example 'posts.**' # => Include related posts, and all the included
   #   posts' related resources, and their related resources, recursively.
   class IncludeDirective
+    attr_reader :options, :hash
```

Comment:
And also `JSONAPI::IncludeDirective#hash` overrides [`Object#hash`](http://ruby-doc.org/core-2.3.1/Object.html#method-i-hash) method that is used for equality matching.